schedule/yast/encryption/activate_encrypted_volume+import_users.yaml

```diff
@@ -22,7 +22,6 @@ schedule:
   - installation/installer_timezone
   - installation/hostname_inst
   - installation/user_import
-  - installation/user_settings_root
```

Comment:
@jknphy Do we need this change, or we should replace it here with the new test module? (`root_simple_pwd.pm`)